src/lib.rs

```diff
@@ -41,13 +41,4 @@ mod protocol;
 mod compression;
 
 pub use self::error::{Error, Result};
-
-trait KafkaClientInternals {
-    fn internal_produce_messages<'a, 'b, I, J>(&mut self,
-                                               ack: client::RequiredAcks,
-                                               ack_timeout: i32,
-                                               messages: I)
-                                               -> Result<Vec<client::TopicPartitionOffset>>
-        where J: AsRef<client::ProduceMessage<'a, 'b>>,
-              I: IntoIterator<Item = J>;
-}
+pub use client_internals::KafkaClientInternals;
```

Comment:
Oh, I see; yeah, I just saw that this was a duplicate, but I failed to see it wasn't public. I'll just remove this line.